include/gatekeeper_log_ratelimit.h

```diff
@@ -74,7 +74,17 @@ void log_ratelimit_state_init(unsigned int lcore_id, uint32_t interval,
  *   - 0: Success.
  *   - Negative on error.
  */
-int rte_log_ratelimit(uint32_t level, uint32_t logtype, const char *format, ...)
+int gatekeeper_log_ratelimit(uint32_t level, uint32_t logtype,
+	const char *format, ...)
+#ifdef __GNUC__
+#if (__GNUC__ > 4 || (__GNUC__ == 4 && __GNUC_MINOR__ > 2))
+	__attribute__((cold))
+#endif
+#endif
+	__attribute__((format(printf, 3, 4)));
+
+int gatekeeper_main_log(uint32_t level, uint32_t logtype,
```

Comment:
Rename `gatekeeper_main_log` to `gatekeeper_log_main` to match `gatekeeper_log_ratelimit`.